aspnet/fundamentals/configuration.rst

```diff
@@ -152,46 +151,46 @@ In addition to using the built-in configuration sources, you can also write your
 Example: Entity Framework Settings
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You may wish to store some of your application's settings in a database, and access them using Entity Framework Core (EF). There are many ways in which you could choose to store such values, ranging from a simple table with a column for the setting name and another column for the setting value, to having separate columns for each setting value. In this example, I'm going to create a simple configuration source that reads name-value pairs from a database using EF.
+You may wish to store some of your application's settings in a database, and access them using Entity Framework Core (EF). There are many ways in which you could choose to store such values, ranging from a simple table with a column for the setting name and another column for the setting value, to having separate columns for each setting value. In this example, we're going to create a simple configuration source that reads name-value pairs from a database using EF.
```

Comment:
This example IMO suffers from an unsolvable chicken-and-egg problem: although you can have EF Core work completely independent form the application's service container I am not sure we want to recommend customers to do that in ASP.NET Core applications. Normally EF Core would only be used after the container is available, but configuration needs to work before that. I wish we just found some other idea of where to store things for an introductory example.